web/embed_components.html

```diff
@@ -282,6 +282,10 @@
 
 <dom-module id="dartpad-dom-output">
   <style>
+    :host [hidden] {
```

Comment:
You _may_ be able to do this instead `:host[hidden]` - it's a bit more specific a selector.